IdentityCore/src/oauth2/MSIDWebviewFactory.m

```diff
@@ -81,11 +81,15 @@ @implementation MSIDWebviewFactory
             default:
                 break;
         }
-    
+
+#if !MSID_EXCLUDE_SYSTEMWV
     return [self systemWebviewFromConfiguration:configuration
                        useAuthenticationSession:useSession
                       allowSafariViewController:allowSafariViewController
                                         context:context];
+#else
```

Comment:
are we returning nil only if it is system webview  for all targets ?

Reply:
Yes, any partner regardless of platform may choose to avoid system webview entirely.  These `!MSID_EXCLUDE_SYSTEMWV` changes are added in a few places where setting it was not working. Other changes that add `#if defined TARGET_OS_VISION && TARGET_OS_VISION` are meant to specifically remove SafariViewController because it is completely removed on visionOS

In this specific case the `[systemWebviewFromConfiguration]` method itself is wrapped in `#if !MSID_EXCLUDE_SYSTEMWV`, but the call to it here was not. It's safer to wrap the call itself as well so we don't call a non-existent method